deployment.rb

```diff
@@ -69,6 +69,8 @@ def load_configuration()
     'read_only'                   => false,
     'ruby_installer'              => rack_env == :development ? 'rbenv' : 'system',
     'root_dir'                    => root_dir,
+    'use_dynamo_tables'           => [:staging, :production].include?(rack_env),
```

Comment:
@wjordan @islemaster FYI that Dynamo is currently configured to be used for the tables API in production after our next deploy. I think this should be ok considering that only Netsim and Applab data blocks depend on the tables API, and that neither Netsim nor Applab have "launched". Let me know if you have concerns, though.